test/spec/lazy-load-spec.js

```diff
@@ -0,0 +1,18 @@
+describe("Lazy Loaded Image", function() {
+  it("should return empty src on Firefox", function() {
+    cl = cloudinary.Cloudinary.new({cloud_name: 'sdk-test'});
+    var img;
+    let options = {loading: "lazy", width: "300"}
+    img = cl.image("sample", {
+      options
+    });
+    cl.handleLazyLoad(img, options);
```

Comment:
@strausr , why is that needed to be called?

Reply:
The method is expecting an img so we're creating one using image()